mkdocs/themes/mkdocs/css/base.css

```diff
@@ -226,24 +240,110 @@ blockquote {
+}
+
+.admonition.note h1, .admonition.note h2, .admonition.note h3,
+.admonition.note h4, .admonition.note h5, .admonition.note h6,
+details.note h1, details.note h2, details.note h3,
+details.note h4, details.note h5, details.note h6 {
+    color: var(--bs-primary-text-emphasis);
+}
+
+.admonition.note a, details.note a {
+    color: var(--bs-primary-text-emphasis);
+}
+
+.admonition.info, details.info {
+    color: var(--bs-info-text-emphasis);
+    background-color: var(--bs-info-bg-subtle);
+    border-color: var(--bs-info-border-subtle);
+}
+
+.admonition.info h1, .admonition.info h2, .admonition.info h3,
+.admonition.info h4, .admonition.info h5, .admonition.info h6,
+details.info h1, details.info h2, details.info h3,
+details.info h4, details.info h5, details.info h6 {
+    color: var(--bs-info-text-emphasis);
+}
+
+.admonition.info a, details.info a {
+    color: var(--bs-info-text-emphasis);
 }
 
-.admonition.warning, details.warning { /* csslint allow: adjoining-classes */
-    color: #7a6032;
-    background-color: #fffae5;
-    border-color: #fbeed5;
+.admonition.warning, details.warning {
+    color: var(--bs-warning-text-emphasis);
+    background-color: var(--bs-warning-bg-subtle);
+    border-color: var(--bs-warning-border-subtle);
 }
 
-.admonition.danger, details.danger { /* csslint allow: adjoining-classes */
-    color: #7f3130;
-    background-color: #fde3e3;
-    border-color: #eed3d7;
+.admonition.warning h1, .admonition.warning h2, .admonition.warning h3,
+.admonition.warning h4, .admonition.warning h5, .admonition.warning h6,
+details.warning h1, details.warning h2, details.warning h3,
+details.warning h4, details.warning h5, details.warning h6 {
+    color: var(--bs-warning-text-emphasis);
 }
 
+.admonition.warning a, details.warning a {
+    color: var(--bs-warning-text-emphasis);
+}
+
+.admonition.danger, details.danger {
+    color: var(--bs-danger-text-emphasis);
+    background-color: var(--bs-danger-bg-subtle);
+    border-color: var(--bs-danger-border-subtle);
+}
+
+.admonition.danger h1, .admonition.danger h2, .admonition.danger h3,
+.admonition.danger h4, .admonition.danger h5, .admonition.danger h6,
+details.danger h1, details.danger h2, details.danger h3,
+details.danger h4, details.danger h5, details.danger h6 {
+    color: var(--bs-danger-text-emphasis);
+}
+
+.admonition.danger a, details.danger a {
+    color: var(--bs-danger-text-emphasis);
+}
+
+.admonition.new, details.new {
+    color: var(--bs-success-text-emphasis);
+    background-color: var(--bs-success-bg-subtle);
+    border-color: var(--bs-success-border-subtle);
+}
+
+.admonition.new h1, .admonition.new h2, .admonition.new h3,
+.admonition.new h4, .admonition.new h5, .admonition.new h6,
+details.new h1, details.new h2, details.new h3,
+details.new h4, details.new h5, details.new h6 {
+    color: var(--bs-success-text-emphasis);
+}
+
+.admonition.new a, details.new a {
+    color: var(--bs-success-text-emphasis);
+}
```

Comment:
So the `new` and `example` admonition styles were previously defined in `extra.css` which unique to MkDocs' documentation. I moved them to the theme because I though they made sense there. After all, new and example are the sorts of blocks one might want in technical documentation, which is MkDocs primary use case. And, they are using two of the otherwise unused alert styles defined by Bootstrap. Guess I don't see why they can't be included with the theme. But if that is what you really want I can move them back. 😕

Reply:
Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/351b87939ddc61bc767e2b869e2a2673fe9af5ae